rust-toolchain

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2024-03-05"
-components = ["rust-src", "rustc-dev", "llvm-tools-preview"]
+channel = "nightly-2024-03-10"
+components = ["rust-src", "rustc-dev", "llvm-tools-preview", "rustfmt"]
```

Comment:
Why add `rustfmt` here?

Reply:
Because we need it to run `cargo fmt` (which is enforced in the CI).

Reply:
Yeah, but the people who only want to compile the project might not want it.
So, I would remove this unless you see a reason to keep it.

Reply:
Hum ok, removing it.